build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+SVGO=$(command -v svgo)
+if [ ! -e "${SVGO}" ]; then
+		echo "ERROR - SVGO not found. Install with npm."
+		echo "    npm install -g svgo"
+		exit 1
+fi
+
+SVGEXPORT=$(command -v svgexport)
+if [ ! -e "${SVGEXPORT}" ]; then
+		echo "ERROR - svgexport not found. Install with npm."
+		echo "    npm install -g svgexport"
+		exit 1
+fi
+
+PROJECT_ROOT=$(pwd)
+BUILD_DIR="$PROJECT_ROOT/build"
+
+#Start from a clean slate
+rm -rf "$BUILD_DIR"
+
+for distro_icon in src/distro-icons/*.svg; do
+	# Create directories for every distro (e.g. 'build/fedora/png' and 'build/fedora/svg')
+	DISTRO_NAME=$(basename -a -s .svg "$distro_icon")
+	PNG_DIR="$BUILD_DIR/$DISTRO_NAME/png"
+	SVG_DIR="$BUILD_DIR/$DISTRO_NAME/svg"
+	mkdir -p "$PNG_DIR"
+	mkdir -p "$SVG_DIR"
+
+	for quickemu_icon in src/quickemu-icons/*.svg; do
```

Comment:
I ran `build.sh` — I think this should be limited to *only* the quickemu icons that have a background. We should enable mixing and matching but include guardrails to avoid explicitly bad combinations.

Reply:
Yeah, this doesn't look that good xD